### DISCUSSION TITLE
How do I modify some Rust item using an autocommand callback?

### DISCUSSION THREAD

**Discussion by dasupradyumna** [noib3/nvim-oxi#207] [Q&A] [answered]
When I try doing this in autocommand callbacks, I get an error telling me that callbacks expect `Fn` closures and I am providing `FnMut` closures. How do I go about achieving this?

**Comment by noib3** [comment, marked as answer]
Use `Function::from_fn_mut()`.

**Reply by dasupradyumna** [reply]
Thanks!
